client/src/components/add-employer-component/addEmployer.jsx

```diff
@@ -1,11 +1,20 @@
-import { useState } from "react";
```

Comment:
We currently can't use the navigation at the bottom of the add employer page
<img width="122" alt="Screenshot 2024-02-05 at 12 30 57 AM" src="https://github.com/uoftblueprint/tces/assets/58835213/ae4e4548-2ea6-4d59-b70c-466ac07d595a">

Reply:
I purposely did this because the bottom pagination navigation didn't support the form requirements before moving onto the next page. from this, i disabled the ability to navigate by the bottom navigation to avoid complexity and make sure that before going onto the next page, all the required fields are filled in